src/platform/mhv/util/helpers.js

```diff
@@ -109,3 +111,104 @@ export const pharmacyPhoneNumber = prescription => {
   }
   return null;
 };
+
+/**
+ * Returns the date and time for file download name
+ * @param {Object} user user object from redux store
+ * @returns the user's name with the date and time in the format John-Doe-M-D-YYYY_hhmmssa
+ */
+export const getNameDateAndTime = user => {
+  const now = new Date();
+  const formattedDate = format(now, 'M-d-yyyy_hhmmaaa'); // Note: `aa` gives lowercase am/pm in some locales
+  return `${user.userFullName.first}-${
+    user.userFullName.last
+  }-${formattedDate}`;
+};
+
+/**
+ * @param {Object} nameObject {first, middle, last, suffix}
+ * @returns {String} formatted timestamp
+ */
+export const formatNameFirstLast = ({ first, middle, last, suffix }) => {
+  let returnName = '';
+
+  let firstName = `${first}`;
+  let lastName = `${last}`;
+
+  if (!first) {
+    return lastName;
+  }
+  if (middle) firstName += ` ${middle}`;
+  if (suffix) lastName += `, ${suffix}`;
+
+  returnName = `${firstName} ${lastName}`;
+
+  return returnName;
+};
+
+/**
+ * Formats the user's date of birth into a long-form date string.
+ *
+ * If a date of birth is present in the user profile, it returns the formatted date
+ * (e.g., "January 1, 1980"). If not, it returns the string "Not found".
+ *
+ * @param {Object} userProfile - The user profile object, typically from Redux state.
+ * @param {string} userProfile.dob - The user's date of birth in ISO string format.
+ * @returns {string} A formatted date string or "Not found" if the DOB is missing.
+ */
+export const formatUserDob = userProfile => {
+  return userProfile?.dob ? formatDateLong(userProfile.dob) : 'Not found';
+};
+
+/**
+ * @param {Error} error javascript error
+ * @param {String} page name of the page sending the error
+ * @returns {undefined}
+ */
+export const sendErrorToSentry = (error, page) => {
+  Sentry.captureException(error);
+  Sentry.captureMessage(
+    `MHV - Medical Records - ${page} - PDF generation error`,
```

Comment:
I don't want this to slow things down, so could we get a follow-up PR to either generalize this `sendErrorToSentry` function or alter the name to specify it is for Medical Records?